concurrent_map.go

```diff
@@ -77,6 +77,7 @@ func (m *ConcurrentMap) Set(key string, value interface{}) {
 		appendKeyToList(key, shard)
 	} else {
 		shard.mapKeys[v.arrayIdx] = ""
+		appendKeyToList(key, shard)
```

Comment:
I think that the code between 76-81 could be replaced with:
```
if ok {
    shard.mapKeys[v.arrayIdx] = ""
}
appendKeyToList(key, shard)
```

Reply:
done